container_pipeline/trigger_build.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+"""
+Trigger build.
+
+This is for creating openshift projects after job passes the linting.
+This also triggers the build after job is created in openshift
+"""
+import json
+import os
+import time
+
+import container_pipeline.utils as utils
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+
+
+def create_project(queue, job, logger):
+    job_name = job.get("job_name")
+    project = utils.get_job_hash(job_name)
```

Comment:
We're storing hash value of the job in the variable `project`. This makes things a bit ambiguous. Can we use a more relevant variable name instead?